types/index.d.ts

```diff
@@ -2574,6 +2574,16 @@ declare namespace math {
      */
     erf<T extends number | MathCollection>(x: T): NoLiteralType<T>
 
+    /**
+     * Compute the Riemann Zeta function of a value using an infinite series
+     * and Riemann's Functional equation.
+     * @param x A real or complex number
+     * @returns The Riemann Zeta of x
+     */
+    riemannZeta<T extends number | Complex | MathCollection>(
```

Comment:
Can you add `BigNumber` to the TypeScript definitions (2x) ? It would be too bad if none of the TypeScript users would know you can actually use BigNumbers ;)